tests/model_fields/test_jsonfield.py

```diff
@@ -1007,7 +1007,7 @@ def test_key_sql_injection(self):
                 False,
             )
         self.assertIn(
-            """."value" -> 'test'' = ''"a"'') OR 1 = 1 OR (''d') = '"x"' """,
```

Comment:
This looks unnecessary. Please revert.

Reply:
It is not, psycopg3 will send `'"x"'::jsonb`

Reply:
:thinking: so, does this test still make sense? Maybe we should use a different string as a value, e.g. add `;` or sth.

Reply:
Since it tests for injection in the key I'd argue that everything after `'"x"'` is not relevant for the test. I might be wrong though.